docs/community/content/involved/conduct/code.en.md

```diff
@@ -108,3 +108,14 @@ The following code of conduct is based on full compliance with [ASF CODE OF COND
    - If a rule's branch is over than `5`, every branch take a new line.
    - Rule name of parser should same with java variable's camel case.
    - Define separate files for every SQL type, file name should consist of `database` + `SQL type` + `Statement`. For example: `MySQLDQLStatement.g4`.
+
+## GitHub Action 规范
+
+- Workflow file name must end with `.yml`
+- Workflow file name must consist with the lowercase of `trigger type-action type`, for example: `nightly-check.yml`. omit trigger type for pull_request, for example: `check.yml`
+- Execution action includes: check, ci, e2e, build
+- Trigger type includes: pull_request (without prefix), nightly
```

Comment:
Trigger type should be the 1st line, Action type 2nd, just keep same sequence with name

Reply:
done